### ISSUE TITLE
[location] Choose Vocab

### ISSUE THREAD

**Issue by SimonGruening** [exc-asia-and-europe/ziziphus#376]
Searching and choosing location terms would be much easier, if you could choose which controlled vocab you'd like to use for the search, like we did for the Subject Set

**Comment by jodevelops** [comment]
= filter result list by its authority source - that would be VIAF and tgn - and others, like "local"?

Apparently, it was agreed (?), that getty terms should be prioritized in the results list (first tgn, then VIAF terms), so there is kind of a filter (sort order) involved already that excludes VIAF from "first sight".

**Comment by SimonGruening** [comment]
Which is pretty inconvenient as all repositories are added from VIAF.

**Comment by MatthiasArnold** [comment]
my suggestion:

* a) default is search "as-is"
* b) add a NEW combo box
    * default value is "All vocabularies"
    * users can select one or multiple entries from that list to only search within the selected vocabularies.

@SimonGruening can you provide a little mockup?

(This combo-box should also appear at all other positions where more than one vocabulary can be used.)

**Comment by SimonGruening** [comment]
I thought maybe like this. Default all tick boxes are chosen and the user needs to un-choose a vocabulary:
![choose_vocab_location](https://cloud.githubusercontent.com/assets/3437957/13434666/f7c8619a-dfd6-11e5-85a3-d637000d0616.png)

**Comment by SimonGruening** [comment]
You mean like it has been done for the subjects?

**Comment by ChiSamurai** [comment]
Yeah, exactly. But I still would prefer one list where you can choose instead of the separation between "vocabs" and "custom vocabs"... here as welle as for the subjects.

**Comment by MatthiasArnold** [comment]
yes, drop-down would be better to handle longer lists. 
it should be possible to select multiple list items. 
combining external and local vocabularies makes much sense, but user access rights need to be checked.
